tests/tests_lite/strategies/launchers/test_multiprocessing.py

```diff
@@ -0,0 +1,94 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
New unit tests added as I saw fit. In PL, the launchers are mainly tested through integration testing, which we can't yet do before all parts have moved to Lite.